pages/properties/gatsby.properties.json

```diff
@@ -1,7 +1,6 @@
 {
     "name": "Gatsby",
-    "creator": "GitHub Actions",
```

Comment:
I would hold drop the author name change.
I dropped the author in https://github.com/paper-spa/starter-workflows/commit/668a20936bcbc6b4cc731df025c25c7a2f87d739

and run `bin/seed pages_repos_codespaces` and `bin/toggle-feature-flag enable pages_build_types`

the author name is gone.

![image](https://user-images.githubusercontent.com/8811182/182485155-7a6f009e-eadb-4780-950c-2ee2819598ae.png)

I think we do not fill the empty in our case, if we drop from here, let's add the case in `github` code first :)

Reply:
😱 that's a good point! On it shortly!

Reply:
You can include author name. It doesn't break anything here.

Reply:
We fixed our UX already to display `GitHub Actions` when `creator` is missing here.

Reply:
So merging this